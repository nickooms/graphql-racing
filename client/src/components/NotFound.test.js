import renderer from 'react-test-renderer';
import React from 'react';

import NotFound from './NotFound';

test('NotFound renders appropriately', () => {
  const component = renderer.create(
    <NotFound />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
