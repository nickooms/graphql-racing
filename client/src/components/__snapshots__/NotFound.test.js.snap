// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders appropriately 1`] = `
<div
  className="NotFound"
>
  404 Not Found
</div>
`;
